src/foam/nanos/notification/email/EmailTemplate.js

```diff
@@ -42,10 +52,23 @@ foam.CLASS({
       documentation: 'Template body',
       view: { class: 'foam.u2.tag.TextArea', rows: 40, cols: 150 },
       javaSetter:
-`body_ = val;
-bodyIsSet_ = true;
-bodyAsByteArray_ = null;
-bodyAsByteArrayIsSet_ = false;`
+        `body_ = val;
```

Comment:
This javaSetter provides no value from what I can tell. Ditch it if that's correct.